src/input/mpegts/satip/satip_frontend.c

```diff
@@ -216,6 +216,14 @@ const idclass_t satip_frontend_class =
       .opts     = PO_ADVANCED,
       .off      = offsetof(satip_frontend_t, sf_pass_weight),
     },
+    {
+      .type     = PT_BOOL,
```

Comment:
This should be tri-state: do-not-use, 0 or 1

Reply:
What do you mean with tri-state? pdids_deladd is set with 0/1, too.

"sd->sd_pids_deladd = 0;"

Reply:
tri-state have the following possible value: not set, disable, enable
I'am not a tvheadend-developer but found this via Google.
I'am very interested in this change because I have two netstream 4c, currently running via minisatip.

Reply:
Of course I understand the tri-state :) What I didn't understand is how to implement the tri-state. I couldn't find a tri-state configuraton type (PT_BOOL, ...). All that I found stored their boolean states as 1 or 0.

Reply:
@F4FNlR  if you want to use the netstream 4c instantly, then you can use my fork...

https://github.com/mpietruschka/tvheadend/tree/release/4.2
https://github.com/mpietruschka/tvheadend